db/scripts/cleanup_scheduled_calls.js

```diff
@@ -5,28 +5,59 @@ async function getDb() {
   dotenv.config();
 
   const pgp = require("pg-promise")({});
-  const connectionURL = process.env.DATABASE_URL;
+  const connectionURL =
+    process.env.NODE_ENV === "test" || process.env.APP_ENV === "test"
+      ? process.env.TEST_DATABASE_URL
+      : process.env.DATABASE_URL;
+
   return pgp({
     connectionString: connectionURL,
-    ssl: { rejectUnauthorized: false },
+    ssl: { rejectUnauthorized: process.env.NODE_ENV === "production" },
```

Comment:
The presence of the ssl key enables / disables SSL in the connection. You will need to remove the key entirely if you want to disable SSL (not just the rejectUnauthorized flag)

Reply:
Done. Seems to work now :slightly_smiling_face: